ansible/playbooks/AdoptOpenJDK_Unix_Playbook/main.yml

```diff
@@ -74,22 +74,31 @@
       when: ansible_distribution != "MacOSX"
     - role: adoptopenjdk_install
       jdk_version: 8
-      when: ansible_distribution != "Alpine"
+      when:
+        - ansible_distribution != "Alpine"
+        - ansible_architecture != "riscv64"
```

Comment:
I can't help feel that these should be within the role, and the role should be kept up-to-date with which ones are available, but that's not really in scope for this PR